brave/browser-laptop
# Replace data-test-id

*The commits have been split for reviews (please feel free to ping me when you would want me to squash some of them). They will be squashed after the reviews. On the 1st commit I replaced selectors inside selectors.js at first.*

Closes #8735

Also this PR fixes errors on about:history and about:bookmarks sortTable errors, which appear on https://travis-ci.org/brave/browser-laptop/jobs/300861587#L5883 for example.

Auditors:

Test Plan:
1. Check the Travis result: https://travis-ci.org/brave/browser-laptop/builds/301021006
2. Make sure there isn't a regression (I confirmed by myself that there isn't one)
3. `npm run test`
4. Make sure there isn't a regression

Submitter Checklist:


Test Plan:


Reviewer Checklist:

Tests
